README.md

```diff
@@ -1,98 +1,187 @@
+
+Every month, there is a **Point Release** that provides bug-fixes to the LTR and bug-fixes as well as new features to the LR.
+
+### Free and Open Source
 
 QGIS is released under the GNU Public License (GPL) Version 2 or above.
 Developing QGIS under this license means that you can (if you want to) inspect
 and modify the source code and guarantees that you, our happy user will always
 have access to a GIS program that is free of cost and can be freely
 modified.
 
-## Supported raster formats include:
-
- * GRASS
- * USGS DEM
- * ArcInfo binary grid
- * ArcInfo ASCII grid
- * ERDAS Imagine
- * SDTS
- * GeoTiff
- * Tiff with world file
- * WMS, WCS
- * PostgreSQL/PostGIS
- * GeoPackage
-
-## Supported vector formats include:
-
- * ESRI Shapefiles
- * PostgreSQL/PostGIS
- * GRASS
- * GeoPackage
- * SpatiaLite
- * [Other OGR supported formats](http://www.gdal.org/ogr_formats.html)
- * MSSQL
- * Oracle
- * WFS
-
-## Supported mesh formats include:
-
- * NetCDF
- * GRIB
- * 2DM
- * [Other MDAL supported formats](https://github.com/lutraconsulting/MDAL#supported-formats)
-
-## Note
+QGIS is part of the Open-Source Geospatial Foundation ([OSGeo](https://www.osgeo.org/)), offering a range of complementary open-source GIS software projects.
+
+## Installing and using QGIS
 
+Precompiled binaries for QGIS are available at [the QGIS.org download page](https://www.qgis.org/en/site/forusers/download.html).
 Please follow the installation instructions carefully.
-After extracting the distribution, please check the
-[documentation](https://qgis.org/en/docs/index.html)
 
-## Help us
-Please submit bug reports using the [QGIS bug tracker](https://github.com/qgis/QGIS/issues/).
+The [building guide](INSTALL.md) can be used to get started with building QGIS from source.
+
+For installation of QGIS Server, see its [getting started documentation](https://docs.qgis.org/testing/en/docs/server_manual/getting_started.html).
+
+### Documentation
+
+A range of 
+[documentation](https://qgis.org/en/docs/index.html) is available. This includes
+
+- [Training Manual](https://docs.qgis.org/latest/en/docs/training_manual/index.html)
+- [QGIS User Guide](https://docs.qgis.org/latest/en/docs/user_manual/index.html)
+- [QGIS Server Guide](https://docs.qgis.org/latest/en/docs/server_manual/index.html)
+- [Documentation Guidelines](https://docs.qgis.org/latest/en/docs/documentation_guidelines/index.html)
+- [PyQGIS Cookbook](https://docs.qgis.org/latest/en/docs/pyqgis_developer_cookbook/index.html)
+- [Developers Guide](https://docs.qgis.org/latest/en/docs/developers_guide/index.html)
+- [Visual Changelog](https://qgis.org/en/site/forusers/visualchangelogs.html)
+
+### Help and support channels
+
+There are several channels where you can find help and support for QGIS:
+
+- Using the QGIS community site at [https://qgis.org](https://qgis.org)
```

Comment:
```suggestion
- Using the QGIS community site at https://qgis.org
```

Reply:
Not sure what this change adds? Parallelism with other links isn't bad. But happy to change if important.

Reply:
Maybe, parallelism could be 
```suggestion
- Using the QGIS community [site](https://qgis.org)
```
Looks weird to me to encapsulate a hyperlink within a hyperlink. Users would anyway read the url in this case so...
bref... no strong opinion.

Reply:
OK. I like that. fixed.

Reply:
Fixed with [QGIS community site](...)